.changeset/fresh-seahorses-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `@page` support to `selector-pseudo-class-disallowed-list`
```

Comment:
```suggestion
Added: `selector-pseudo-class-disallowed-list` support for `@page`
```

We typically lead with the rule name so that changes to a rule are grouped in the changelog.

Reply:
```suggestion
Added: `selector-pseudo-class-disallowed-list` now checks `@page` at-rules and supports `@page` pseudo-classes
```

copied from https://github.com/stylelint/stylelint/blob/dacc742d1ae55a9b6cef2035132c968d9057d91d/CHANGELOG.md?plain=1#L1409